python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/monitor-server-radius.j2

```diff
@@ -0,0 +1,25 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - monitor server radius #}
+{% if monitor_server_radius is arista.avd.defined %}
+!
+monitor server radius
+{%     if monitor_server_radius.service_dot1x is arista.avd.defined(true) %}
+   service dot1x
+{%     endif %}
+{%     if monitor_server_radius.probe.interval is arista.avd.defined %}
+   probe interval {{ monitor_server_radius.probe.interval }} seconds
+{%     endif %}
+{%     if monitor_server_radius.probe.threshold_failure is arista.avd.defined %}
+   probe threshold failure {{ monitor_server_radius.probe.threshold_failure }}
+{%     endif %}
+{%     if monitor_server_radius.probe.method is arista.avd.defined("status-server") %}
+   probe method status-server
+{%     elif monitor_server_radius.probe.method is arista.avd.defined("access-request") %}
+{%         set access_request = monitor_server_radius.probe.access_request %}
+   probe method access-request username {{ access_request.username }} password {{ access_request.password_type }} {{ access_request.password }}
```

Comment:
```suggestion
   probe method access-request username {{ access_request.username }} password {{ access_request.password_type | arista.avd.default("0") }} {{ access_request.password }}
```

Reply:
In all previous implementations, we have default password_type as 7 as we are accepting only type 7 passwords. If user wants clear text password he needs to choose password type 7. Reference schemas `tacacs_servers`, `monitor_telemetry_influx`.